specfile/tags.py

```diff
@@ -0,0 +1,599 @@
+    "Modularitylabel",
+    "N",
+    "Name",
+    "Nevr",
+    "Nevra",
+    "Nopatch",
+    "Nosource",
+    "Nvr",
+    "Nvra",
+    "O",
+    "Obsoleteflags",
+    "Obsoletename",
+    "Obsoletenevrs",
+    "Obsoletes",
+    "Obsoleteversion",
+    "Optflags",
+    "Orderflags",
+    "Ordername",
+    "Orderversion",
+    "Origbasenames",
+    "Origdirindexes",
+    "Origdirnames",
+    "Origfilenames",
+    "Os",
+    "P",
+    "Packager",
+    "Patch",
+    "Patchesflags",
+    "Patchesname",
+    "Patchesversion",
+    "Payloadcompressor",
+    "Payloaddigest",
+    "Payloaddigestalgo",
+    "Payloaddigestalt",
+    "Payloadflags",
+    "Payloadformat",
+    "Pkgid",
+    "Platform",
+    "Policies",
+    "Policyflags",
+    "Policynames",
+    "Policytypes",
+    "Policytypesindexes",
+    "Postin",
+    "Postinflags",
+    "Postinprog",
+    "Posttrans",
+    "Posttransflags",
+    "Posttransprog",
+    "Postun",
+    "Postunflags",
+    "Postunprog",
+    "Prefixes",
+    "Prein",
+    "Preinflags",
+    "Preinprog",
+    "Prereq",
+    "Pretrans",
+    "Pretransflags",
+    "Pretransprog",
+    "Preun",
+    "Preunflags",
+    "Preunprog",
+    "Provideflags",
+    "Providename",
+    "Providenevrs",
+    "Provides",
+    "Provideversion",
+    "Pubkeys",
+    "R",
+    "Recommendflags",
+    "Recommendname",
+    "Recommendnevrs",
+    "Recommends",
+    "Recommendversion",
+    "Recontexts",
+    "Release",
+    "Removepathpostfixes",
+    "Removetid",
+    "Requireflags",
+    "Requirename",
+    "Requirenevrs",
+    "Requires",
+    "Requireversion",
+    "Rpmversion",
+    "Rsaheader",
+    "Sha1header",
+    "Sha256header",
+    "Sig_base",
+    "Siggpg",
+    "Sigmd5",
+    "Sigpgp",
+    "Sigsize",
+    "Size",
+    "Source",
+    "Sourcepackage",
+    "Sourcepkgid",
+    "Sourcerpm",
+    "Suggestflags",
+    "Suggestname",
+    "Suggestnevrs",
+    "Suggests",
+    "Suggestversion",
+    "Summary",
+    "Supplementflags",
+    "Supplementname",
+    "Supplementnevrs",
+    "Supplements",
+    "Supplementversion",
+    "Transfiletriggerconds",
+    "Transfiletriggerflags",
+    "Transfiletriggerin",
+    "Transfiletriggerindex",
+    "Transfiletriggername",
+    "Transfiletriggerpostun",
+    "Transfiletriggerpriorities",
+    "Transfiletriggerscriptflags",
+    "Transfiletriggerscriptprog",
+    "Transfiletriggerscripts",
+    "Transfiletriggertype",
+    "Transfiletriggerun",
+    "Transfiletriggerversion",
+    "Triggerconds",
+    "Triggerflags",
+    "Triggerin",
+    "Triggerindex",
+    "Triggername",
+    "Triggerpostun",
+    "Triggerprein",
+    "Triggerscriptflags",
+    "Triggerscriptprog",
+    "Triggerscripts",
+    "Triggertype",
+    "Triggerun",
+    "Triggerversion",
+    "Url",
+    "V",
+    "Vcs",
+    "Vendor",
+    "Verbose",
+    "Verifyscript",
+    "Verifyscriptflags",
+    "Verifyscriptprog",
+    "Veritysignaturealgo",
+    "Veritysignatures",
+    "Version",
+    "Xpm",
+}
+
+
+class Comment:
```

Comment:
I guess this could be implemented (and slightly simplified) using a [dataclass](https://docs.python.org/3/library/dataclasses.html) since it basically only stores the prefix and text which would remove the need for explicitly defining repr/str

Reply:
> which would remove the need for explicitly defining repr/str

For `__repr__()`, yes, but not for `__str__()`.

I'm not sure, is it worth it?

Reply:
Good point, it probably wouldn't really make much difference, constructor and `__repr__` could be removed but that's about it and it wouldn't simplify it a lot since it's only 2 attributes. I am fine with keeping it the way it is now